db/1.8/modules/ROOT/pages/database-connector-connection.adoc

```diff
@@ -0,0 +1,393 @@
+. In the *Database Config* window, for *Connection* select **Oracle Connection**.
+. Click *Configure* to add the required Oracle JDBC driver and select either: +
++
+* *Use local file* +
+Install the library using a local file.
+* *Add Maven dependency* +
+Install a Maven dependency to add to the project.
++
+. Set *Host* to `192.168.99.100`, *Port* to `1521`, *User* to `system`, *Password* to `oracle` and *Instance* to `xe`.
+. On the *Transactions* tab, optionally specify the transaction isolation, or XA transactions when connecting to the database.
+. On the *Advanced* tab, optionally specify connection pooling and reconnection information, including a reconnection strategy.
+. Click *Test Connection* to confirm that Mule can connect to the database.
+. Click *OK*.
+
+The following screenshot shows the *Oracle Connection* configuration in Studio:
+
+.Oracle Connection configuration
+image::database-oracle-connection.png[Oracle Connection configuration]
+
+In the XML editor, the configuration looks like this:
+
+[source,xml,linenums]
+----
+<db:config name="Database_Config">
+  <db:oracle-connection
+  host="192.168.99.100"
+  port="1521" instance="xe"
+  user="system"
+  password="oracle" />
+</db:config>
+----
+
+== Configure the JDBC Driver
+
+After you configure the connections, you can add the JDBC driver. The following example shows how to add a driver for the MySQL database:
+
+. In Studio, you can add the driver using any of the following options:
+* In the *Database Config* window, click *Configure*, select *Add Maven dependency*, and pick the dependency:
++
+.Pick Maven dependency
+image::database-dependency.png[Pick Maven dependency]
++
+* Add the dependency directly in your project's `pom.xml` file:
++
+[source,xml,linenums]
+----
+<dependency>
+    <groupId>mysql</groupId>
+    <artifactId>mysql-connector-java</artifactId>
+    <version>5.1.48</version>
+</dependency>
+----
++
+. Through your `pom.xml` file, make Mule expose the driver dependency to the Database Connector:
++
+[source,xml,linenums]
+----
+<build>
+    <plugins>
+        <!-- Only used to declare the shared libraries-->
+        <plugin>
+            <groupId>org.mule.tools.maven</groupId>
+            <artifactId>mule-maven-plugin</artifactId>
+            <version>1.0.0</version>
+            <configuration>
+                <sharedLibraries>
+                    <sharedLibrary>
+                        <groupId>mysql</groupId>
+                        <artifactId>mysql-connector-java</artifactId>
+                    </sharedLibrary>
+                </sharedLibraries>
+            </configuration>
+        </plugin>
+    </plugins>
+</build>
+----
+
+== Configure Connection Pools
+
+Establishing connections is costly in terms of connectors, relational database management system (RDBMS) network, and processing overhead. Database Connector supports the *Pooling profile* parameter that enables you to build a connection pool to provide several connections.
```

Comment:
```suggestion
Establishing connections is costly in terms of connectors, relational database management system (RDBMS) network, and processing overhead. Database Connector supports the `Pooling profile` parameter that enables you to build a connection pool to provide several connections.
```
I went ahead and changed this because the sentence specifies "parameter".

But I think you GLOBALLY should change the boldface...:
- connection types to plain font, and lowercase
- configuration types and names to plain font
- syntax (parameters, options, files, directories...) to monospace font

Leave references to UI labels in bold: windows, tabs, fields, and so on.

Reply:
I've gone back to not marking these because there are so many.